### ISSUE TITLE
Edit link is missing a space

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyone#386]
**Describe the bug**
Tested with NVDA screen reader on Windows 10.
The edit link and post title is missing a space and is read out as
_editpost title_

**To Reproduce**
1. Start the screen reader
2. Listen to the edit link / read the speech viewer.

**Expected behavior**
The text should be read out as separate words.

**Comment by aristath** [comment]
I have a solution to suggest...
We could just remove the edit links.
They were initially added to WordPress themes as a shortcut to quickly edit posts, but that was before the "Edit Post" link was added to the adminbar. Other than tradition I really see no reason for their existence anymore.

**Comment by carolinan** [comment]
During Fridays bug scrub, it was decided that the edit link should be kept.
